src/ImportExport.ts

```diff
@@ -177,6 +178,7 @@ export const exportSynergism = async () => {
             })
 
             clipboard.on('error', () => {
+                DOMCacheGetOrSet('exportinfo').textContent = 'Export failed! DM Ant God'
```

Comment:
```suggestion
                DOMCacheGetOrSet('exportinfo').textContent = 'Export failed!'
```

Errors will likely only come from mobile safari, which limits how copy/paste works. Nothing we can do if it occurs, so no point in having someone dm @Pseudonian over this

Reply:
Was just being cheeky. Maybe should have been Pray to the Ant God. Or Prey of the Ant God. But fair point, would be smart to avoid actual PMs